### PR TITLE
fix(button): loading buttons still shows labels

### LINKED ISSUE #2288: [dropdown] Item label/icon is still visible for loading dropdown
The dropdown item label/icon is still visible when the dropdown gets the `loading` class

Please see this fiddle for reference: https://jsfiddle.net/5pro4jvc/1/ (just select any item)

### PR DESCRIPTION
**Pull request by lubber-de**
## Decription
Whenever a `loading button` contains any kind of label it was still shown

## Testcase
- Select anything from the dropdown (it will get the loading state then)
- Remove CSS to see issue
https://jsfiddle.net/lubber/mrza0234/9/

## Broken
image

## Closes
#2288

### CODE REVIEW

File: src/definitions/elements/button.less [hunk 1]
```diff
@@ -169,19 +169,24 @@
 
     box-shadow: 0 0 0 1px transparent;
   }
-}
-& when (@variationButtonLabeledIcon){
-  .ui.labeled.icon.loading.button .icon {
+  .ui.ui.loading.button .label {
```
Comment (ivangomes) [inline src/definitions/elements/button.less:172]: .ui.ui.loading.button

.ui.ui <-- is this a typo?

Comment (exoego) [reply src/definitions/elements/button.less:172]: This is a CSS technique to increase specificity of the selector
https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity

Comment (lubber-de) [reply src/definitions/elements/button.less]: I even had to increase it again to make it work properly when compiled by 6d249925aa0695b344c5f2e92417f8b76ed12237 :wink:

**Review by exoego — APPROVED** [review]
LGTM

**Pull request merged**